packages/hierarchies/src/hierarchies/imodel/FilteringHierarchyDefinition.ts

```diff
@@ -21,6 +21,7 @@ import {
 } from "./IModelHierarchyDefinition";
 import { ProcessedGroupingHierarchyNode, ProcessedHierarchyNode, SourceHierarchyNode, SourceInstanceHierarchyNode } from "./IModelHierarchyNode";
 import { defaultNodesParser } from "./TreeNodesReader";
+import { Dictionary } from "@itwin/core-bentley";
```

Comment:
Imports from other packages should be listed before imports from local modules. I suggest using https://marketplace.visualstudio.com/items?itemName=mike-co.import-sorter for that.

Reply:
I had opened presentation repo in co-space, for some reason import sorter does not work in co-space. Fixed the order